app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -361,3 +361,7 @@ a.btn-info span {
 .gridster .gs-w {
   z-index: auto !important;
 }
+
+#history a .glyphicon {
```

Comment:
This would be nice to avoid, but there doesn't seem to be any applicable style that makes the icon dark, while keeping the text the default link color.